Swerve2022/src/main/java/frc/robot/Constants.java

```diff
@@ -12,25 +14,38 @@
  */
 public class Constants
 {
-  /**
-   * public static final class DriveConstants {
-   * public static final int kLeftMotor1Port = 0;
-   * public static final int kLeftMotor2Port = 1;
-   * public static final int kRightMotor1Port = 2;
-   * public static final int kRightMotor2Port = 3;
-   * }
-   */
+  public static final class Falcon500
+  {
+    public static int          kMaxRPM               = 6380;             // free speed for Falcon 500 motor
+    public static final double kEncoderCPR           = 2048;             // CPR is 2048 from Falcon 500 Manual
+    public static final int    kTalonReqVersion      = ((22 * 256) + 0); // Talon version is 22.0
+    public static final int    kPigeonReqVersion     = ((22 * 256) + 0); // Pigeon IMU version is 22.0
+
+    // Input current limit settings
+    public static final double kSupplyCurrentLimit   = 45.0;  // Default supply current limit (after trigger)
+    public static final double kSupplyTriggerCurrent = 45.0;  // Trigger current that will cause limiting
+    public static final double kSupplyTriggerTime    = 0.001; // Time duration of trigger that will causing limiting
+
+    // Output current limit settings
+    public static final double kStatorCurrentLimit   = 80.0;  // Default supply current limit (after trigger)
+    public static final double kStatorTriggerCurrent = 80.0;  // Default trigger current that will cause limiting
+    public static final double kStatorTriggerTime    = 0.001; // Default time duration of trigger that will causing limiting
+  }
+
   public static final class DTConsts
   {
+    public static final int    kLFDrive1CANID         = 1;
+    public static final int    kLFSteer2CANID         = 2;
+    public static final int    kRFDrive3CANID         = 3;
+    public static final int    kRFSteer4CANID         = 4;
+
+    public static final int    kLRDrive5CANID         = 5;
+    public static final int    kLRSteer6CANID         = 6;
+    public static final int    kRRDrive7CANID         = 7;
+    public static final int    kRRSteer8CANID         = 8;
 
-    public static final int kLFDrive1CANID = 1;
-    public static final int kLFSteer2CANID = 2;
-    public static final int kRFDrive3CANID = 3;
-    public static final int kRFSteer4CANID = 4;
+    public static final double kWheelDiaMeters        = Units.inchesToMeters(4.0); // 4in (39.37 in/meter)
+    public static final double kEncoderMetersPerCount = (kWheelDiaMeters * Math.PI) / (Falcon500.kEncoderCPR) / 6.75;
```

Comment:
Let's make another named constant for the gear ratio, so the next person knows what that 6.75 magic number is for. I think we did that in the Offseason2022 project.

Reply:
I'm not quite sure what I need to change for having the steer/turn motors constants and object naming matching.

Reply:
Just change the name of the kXXSteerNCANID literals to be kXXTurnNCANID (where XX is LF, RF, etc. and N is the CAN ID value). 

Then when SwerveModule is called, the parameter name will match the constant being passed.

Reply:
Looks good now.